tests/Extensions/Hosting.Services.Web.UnitTests/HostBuilderExtensionsTests.cs

```diff
@@ -44,8 +44,8 @@ public async Task BuildStatelessWebService_RegisterListeners()
 				"someService2",
 				new WebEndpointInfo[]
 				{
-					WebEndpointInfo.CreateHttp(httpListener1.name),
-					WebEndpointInfo.CreateHttp(httpListener2.name)
+					new WebEndpointInfo(httpListener1.name, null),
+					new WebEndpointInfo(httpListener2.name, null)
```

Comment:
please label the param name when just passing null - nameOfParam: null

Reply:
updated, thanks